addons/website/static/src/snippets/s_tabs/002.scss

```diff
@@ -51,8 +51,13 @@
             .nav-link {
                 margin-bottom: -1px;
 
+                &.active {
+                    color: inherit;
+                }
+                
                 &:not(.active) {
                     border: 0;
+                    color: inherit;
```

Comment:
- color inherit if active + color inherit if not active, why not color inherit always then?
- also, is this a very specific issue to s_tabs? Sure it's not a generic problem of o_cc classes?

Reply:
@xlu-odoo If it is finished and ready for review, can you answer my second question here? Thanks :pray: